### DISCUSSION TITLE
Is there a way to download the videos for a specific description in a period of time

### DISCUSSION THREAD

**Discussion by davidyushen** [davidteather/TikTok-Api#834] [Q&A] [answered]
Hi, is it possible to let the api download the videos with specific descriptions such as description include "covid"?

**Comment by Jheesbrough** [comment, marked as answer]
You can read the descriptions of videos when you search for them and only download the ones you want.
using `video.as_dict['desc']` will give you the description

**Comment by Popesites** [comment]
There's certainly a way to do so. in the tiktok app, there is a way to filter results by last day, 7 days, 30 days etc. This package doesn't do it yet, but it needs to support it.
